stdpopsim/ext/selection.py

```diff
@@ -76,7 +76,7 @@ class DrawMutation(ExtendedEvent):
     """
 
     time = attr.ib(type=float)
-    mutation_type_id = attr.ib(type=int)
+    mutation_id = attr.ib(type=str)
```

Comment:
this change looks fine to me. I'd want to check with @mufernando, who i believe agonized a bit over `mutation_type_id`s before, but everything seems to be working on my end